packages/rich-text/src/plugins/EmbeddedResourceBlock/LinkedResourceBlock.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
```

Comment:
A lot of the code here is copied directly from the LinkedEntityBlock. Considering they are so similar could you please look into extracting out the shared logic please?

Reply:
Done